### ISSUE TITLE
[White Label- Card Bingsports]: Narrator focus is moving to decorative image.

### ISSUE THREAD

**Issue by Kusum199** [microsoft/BotFramework-WebChat#5717]
### What is the PWD impact?

Screen reader users encounter irrelevant content when navigating, causing confusion and unnecessary interruptions. Decorative images being focusable disrupt reading flow and make the UI feel noisy and difficult to navigate.

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

Others or unrelated

### How to reproduce the issue?

Pre-Requisite:
Turn on Narrator

Repro Steps
1. Open URL: https://compulim.github.io/webchat-loader/ and sign in using valid credentials.
2. Navigate to web chat page and select 4.18.1-main.20240927.ea7a875 (30/9/2024) version and then select mockBot link under present label and then select open web chat in new window link at bottom.
3.Navigate to white label experience chat page and type "Card Bingsports" word in type your message text box.
4. Response will appear.
5. Navigate to the image in the response.
6. Observe an issue that Narrator focus is moving to decorative image. 

### What do you expect?

The element should be marked correctly as decorative so that Narrator does not move focus to it.

### What actually happened?

Narrator focus is moving to decorative image. 

### Do you have any screenshots or recordings to repro the issue?

https://github.com/user-attachments/assets/4fd52010-295d-41ad-92f5-3ac8eb9e0da8

### WCAG reference

https://www.w3.org/WAI/WCAG21/Understanding/non-text-content

### Adaptive Card JSON

```json

```
